app/ide-desktop/lib/dashboard/src/layouts/Start.tsx

```diff
@@ -0,0 +1,49 @@
+/** @file Home screen. */
+import * as React from 'react'
+
+import * as textProvider from '#/providers/TextProvider'
+
+import Samples from '#/layouts/Samples'
+import WhatsNew from '#/layouts/WhatsNew'
+
+import * as aria from '#/components/aria'
+import * as ariaComponents from '#/components/AriaComponents'
+
+// ============
+// === Home ===
+// ============
+
+/** Props for a {@link Home}. */
+export interface HomeProps {
+  readonly createProject: (templateId?: string | null, templateName?: string | null) => void
+}
+
+/** Home screen. */
+export default function Home(props: HomeProps) {
```

Comment:
If it's a dialog, why it's in layouts? Also the name doesn't represent tha this is a dialog component